src/userinfo/__tests__/ioUserInfoClient.test.ts

```diff
@@ -0,0 +1,71 @@
+import * as t from "io-ts";
+import * as E from "fp-ts/Either";
+import * as r from "../../__tests__/utils/records";
+import * as p from "../../__tests__/utils/phonies";
+
+const token = "token";
+
+describe("makeIOUserInfoClient", () => {
+  describe("findUserByFederationToken", () => {
+    it("should return a user info", async () => {
+      const { service, mockClient } = p.makeService();
+
+      const functionRecorded = mockClient.getUserIdentity.mockReturnValueOnce(
+        Promise.resolve(
+          t.success({
+            status: 200,
```

Comment:
In your opinion should we move also this `struct` into `phonies` or `records` files?

Reply:
Yeah, but can we keep here for now?